gradient/commands/datasets.py

```diff
@@ -676,3 +685,94 @@ def update_status():
                         for pre_signed in pre_signeds:
                             update_status()
                             pool.put(self._delete, url=pre_signed.url)
+
+
+class ImportDatasetCommand(BaseDatasetsCommand):
+    def create_secret(self, key, value, expires_in=86400):
+        client = api_sdk.clients.SecretsClient(
+            api_key=self.api_key,
+            logger=self.logger,
+            ps_client_name=CLI_PS_CLIENT_NAME,
+        )
+
+        response = client.ephemeral(key, value, expires_in)
+        return response
+
+
+    def get_command_string(self, url):
+        return "%s %s /data/output" % (IMPORTER_COMMAND, url)
+
+
+    def get_command(self, url, http_auth):
+        
+        if url.scheme == 'git' or url.scheme == 's3':
+            return self.get_command_string(url.geturl())
+        
+        if url.scheme == 'https' or url.scheme == 'http':
+            if not http_auth:
+                return self.get_command_string(url.geturl())
+            http_auth_url = "https://${{HTTP_AUTH}}@%s" % url.path
+            return self.get_command_string(http_auth_url)
+
+        return ""
+    
+    def get_env_vars(self, url, secrets):
+        # http_auth, access_key, secret_key
+
+        if url.scheme == 'git':
+            return ""
+
+        if url.scheme == 's3' and secrets[S3_ACCESS_KEY] and secrets[S3_SECRET_KEY]:
```

Comment:
if url.scheme == 's3' and S3_ACCESS_KEY in secrets and S3_SECRET_KEY in secrets: